CMakeLists.txt

```diff
@@ -26,13 +26,16 @@ target_link_libraries(${PROJECT_NAME}_solver ${catkin_LIBRARIES} ${orocos_kdl_LI
 
 add_library(joint_state_listener src/joint_state_listener.cpp)
 target_link_libraries(joint_state_listener ${PROJECT_NAME}_solver ${orocos_kdl_LIBRARIES})
+target_compile_options(joint_state_listener PUBLIC -std=c++11)
```

Comment:
Looks good to me.

If I understand correctly, `target_compile_options()` with `PUBLIC` means the option propagates to targets in other projects that depend or link against `joint_state_listener` as long as the target from this project is exported. However I don't see any c++11 features in the api on the kinetic-devel branch, so It doesn't seem like the c++11 option needs to be exported. Also I don't see anything exporting/installing CMake targets. Maybe catkin exports the targets automatically from the `catkin_package` macro? Regardless `joint_state_listener` isn't listed when that macro is called.